### ISSUE TITLE
plugin-meeting: meeting object exists after meeting end

### ISSUE THREAD

**Issue by arun3528** [webex/webex-js-sdk#1560]
## Expected Behavior

We are seeing meeting object not getting deleted on meeting end
Need to update syncMeeting to update the store . Currently it just checks for the meeting and udpates only if a meeting exist but not when locus == [] deleted all the existing meeting 

## Possible Solution (we welcome any logical suggestion)

Example:  
```
let redirect_uri = `${window.location.protocol}//${window.location.host}`;
if (window.location.pathname) {
  redirect_uri += window.location.pathname;
}
```
## Steps to Reproduce

1.
2.
3.
4.

## Environment Details

* SDK Version
* Node/Browser Version
* NPM Version

**Comment by adamweeks** [comment]
@arun3528 is this issue related to your PR #1575?

**Comment by adamweeks** [comment]
Closed via #1575
